Pipfile

```diff
@@ -15,7 +15,7 @@ wheel = "*"
 # 2.7.0, < 2.8.0". The latest autopep8 specifies "pycodestyle >= 2.8.0". This
 # conflict cannot be resolved. Pin the version to resolve this.
 autopep8 = "<=1.5.7"
-importlib-metadata = "*"
+importlib-metadata = "<5.0"
```

Comment:
Looks like there is a breaking change in https://importlib-metadata.readthedocs.io/en/latest/history.html#v5-0-0.